client/state/stats/lists/utils.js

```diff
@@ -189,8 +189,14 @@ export function getChartLabels( unit, date, localizedDate ) {
 		const isWeekend = 'day' === unit && ( 6 === dayOfWeek || 0 === dayOfWeek );
 		const labelName = `label${ unit.charAt( 0 ).toUpperCase() + unit.slice( 1 ) }`;
 		const formats = {
-			day: 'MMM D',
-			week: 'MMM D',
+			day: translate( 'MMM D', {
+				context: 'moment format string',
+				comment: 'This specifies a day for stats.',
+			} ),
+			week: translate( 'MMM D', {
+				context: 'moment format string',
+				comment: 'This specifies a week for stats.',
```

Comment:
It seems the same comment as `day` would apply here because the value is for a single day (start of the week)

Reply:
Thanks, I've changed the context to separate them.